src/sentry/static/sentry/app/views/alerts/rules/index.tsx

```diff
@@ -104,6 +134,42 @@ class AlertRulesList extends AsyncComponent<Props, State & AsyncComponent['state
     return this.renderBody();
   }
 
+  renderFilterBar() {
+    const {teams} = this.props;
+    const {filteredTeams} = this.state;
+    const teamIds = teams.map(({id}) => id);
+    return (
+      <FilterWrapper>
+        <Filter
+          header={t('Team')}
+          onFilterChange={this.handleChangeFilter}
+          filterList={teamIds}
+          selection={filteredTeams}
+        >
+          {({toggleFilter}) => {
+            return (
+              <List>
+                {teams.map(({id, name}) => (
+                  <ListItem
+                    key={id}
+                    isChecked={filteredTeams.has(id)}
+                    onClick={event => {
+                      event.stopPropagation();
+                      toggleFilter(id);
+                    }}
+                  >
+                    <TeamName>{name}</TeamName>
+                    <CheckboxFancy isChecked={filteredTeams.has(id)} />
+                  </ListItem>
+                ))}
+              </List>
+            );
+          }}
```

Comment:
```suggestion
          {({toggleFilter}) => (
              <List>
                {teams.map(({id, name}) => (
                  <ListItem
                    key={id}
                    isChecked={filteredTeams.has(id)}
                    onClick={event => {
                      event.stopPropagation();
                      toggleFilter(id);
                    }}
                  >
                    <TeamName>{name}</TeamName>
                    <CheckboxFancy isChecked={filteredTeams.has(id)} />
                  </ListItem>
                ))}
              </List>
            )
          }
```
Can be simplified like this